ArduPlane/system.cpp

```diff
@@ -445,80 +445,3 @@ int8_t Plane::throttle_percentage(void)
     }
     return constrain_int16(throttle, -100, 100);
 }
-
-// update the harmonic notch filter center frequency dynamically
-void Plane::update_dynamic_notch()
-{
-    if (!ins.gyro_harmonic_notch_enabled()) {
-        return;
-    }
-    const float ref_freq = ins.get_gyro_harmonic_notch_center_freq_hz();
-    const float ref = ins.get_gyro_harmonic_notch_reference();
-
-    if (is_zero(ref)) {
-        ins.update_harmonic_notch_freq_hz(ref_freq);
-        return;
-    }
-
-    switch (ins.get_gyro_harmonic_notch_tracking_mode()) {
-        case HarmonicNotchDynamicMode::UpdateThrottle: // throttle based tracking
-            // set the harmonic notch filter frequency approximately scaled on motor rpm implied by throttle
-#if HAL_QUADPLANE_ENABLED
```

Comment:
You've lost this check in the move to AP_Vehicle

Reply:
its done via the nullptr check on the AP::motors singleton.

Reply:
ok!